public/app/app.ts

```diff
@@ -99,7 +99,11 @@ export class GrafanaApp {
     setMarkdownOptions({ sanitize: !config.disableSanitizeHtml });
 
     standardEditorsRegistry.setInit(getStandardOptionEditors);
-    standardFieldConfigEditorRegistry.setInit(getStandardFieldConfigs);
+    standardFieldConfigEditorRegistry.setInit(
+      getStandardFieldConfigs({
+        getVariables: getTemplateSrv().getVariables.bind(getTemplateSrv()),
```

Comment:
do the standard editor need access to template vars? Should we allow using those in i.e. display name?